components/dash-table/tests/selenium/test_sizing_x.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from test_sizing import on_focus
+
+from utils import (
+    basic_modes,
+    generate_mock_data,
+)
+
+
+@pytest.mark.parametrize("props", basic_modes)
+@pytest.mark.parametrize(
+    "data_fn",
+    [generate_mock_data],
+)
+def test_szng004_on_focus(test, props, data_fn):
```

Comment:
The `data_fn` parameter can be removed as there is now only one.

Reply:
I am not super familiar with these `pytest`. 
Also I'd prefer to keep the changes in each PR as minimal as possible. 
So if that's non-blocking, I'd suggest we merge and then possibly address extra changes in a separate PR?

Reply:
@archmoj I’d say this is a decent opportunity to learn a bit about `pytest`. @T4rk1n is right, the obvious thing to do here is just remove `data_fn` and it’s associated `@pytest.mark.parametrize` decorator, and move `generate_mock_data` into its place in the function body. This is sufficiently related to what you’re doing in this PR that it should be done here, not in a separate PR.